src/mysql_protocol.erl

```diff
@@ -115,14 +115,30 @@ ping(SockModule, Socket) ->
 query(Query, SockModule, Socket, Timeout) ->
     Req = <<?COM_QUERY, (iolist_to_binary(Query))/binary>>,
     SeqNum0 = 0,
-    {ok, _SeqNum1} = send_packet(SockModule, Socket, Req, SeqNum0),
-    fetch_query_response(SockModule, Socket, Timeout).
+    {ok, SeqNum1} = send_packet(SockModule, Socket, Req, SeqNum0),
+    QryRsp = fetch_query_response(SockModule, Socket, Timeout),
+    case QryRsp of
+    {ok, [#load_local_file{filename = FileName}]} ->
```

Comment:
Indentation after `of` please.

The naming of `QryRsp` is a bit obscure, compared to the other variable names used in this file. How about `Response` or `Resp`? No need to mention that it is from a query, since we are a function called `query`.